src/lib/expansion/expansion-panel-header.scss

```diff
@@ -19,6 +19,19 @@
   &:not([aria-disabled='true']) {
     cursor: pointer;
   }
+
+  // If the panel, that the header is in, is inside an element being animated by Angular,
+  // the default behavior is to block all child animations. This means that the header's
+  // height won't kick in until the parent animation is done. We add a fallback to the
+  // header's default height. If the animations aren't blocked, these styles will be overridden
+  // by the inline styles from the animations module.
+  .ng-animating .mat-expansion-panel & {
+    min-height: 48px;
```

Comment:
This assumes that the height of the panels is going to be `48px` and `64px`.  Does this account for if someone overrides those values using the `@Input`s in the panel header?

Reply:
It doesn't account for the case where people are setting it through the input, but I wasn't able to think of a better way to handle it.

Reply:
I think we might need to reassess if we can try to do the fix from #13888 again.

I think the "correct" solution is for the containing components to allow the animations rather than the components which don't necessarily know what components they will be in.

Because While this is something that we are seeing in expansion-panel, the issue is actually in things like dialog/tabs and if someone has a custom component doing animations they will have the same issue we are seeing.

Reply:
In regards to #13888  and the revert of it in #14017,

> This change broke an app inside of Google where tabs appear inside of a
dialog. The tabs get stuck in a weird frozen animation state.

Is there a test to make sure apps like the internal one mentioned above do not get caught in weird animation state?

I want to contribute to solving #13870 in any way I am able, just not sure how to test this type of fix/regression.

Reply:
@virtuoushub Unfortunately there is not a test to check it because we haven't been able to isolate it outside of the app for a reproduction.  We haven't been able to create a test to check the situation in which it occurs, but with the change from #13888 the failure reproduced in the app with perfect consistency.

I am working with @matsko on sorting it out, but unfortunately we just haven't been able to sort it out yet.

Reply:
Thanks for the update. As @Teamop [mentioned](https://github.com/angular/material2/pull/11778#issuecomment-442423849), whatever fix is decided on, does it relate to #11765 and it's respective PR #11778?